CONTRIBUTING.md

```diff
@@ -164,60 +149,26 @@ Currently the following generators are provided
-
-### Code Consistency
-
-- All files and folders under your package's `src/components` should name with PascalCase except `index.js` files
-- If you need a constant file, it should be called `{Component_Name}Constants.js` (Component_Name with PascalCase)
-- Each component should treat as a standalone package and live under its own folder. If you think your component can be made completely independent from other components, create it as a new package. This reduces the dependency tree and improves performance downstream. It also makes exported bundles lighter.
-- Single file per component with **default export**
-- Avoid using the bindMethods syntax for attaching methods to a class. Instead use class properties for example `testMethod = () => { return 'test'}`
-- When component is a set of components (e.g., ListGroup and ListGroupItem),
-  they should live in the same folder named on the parent component (e.g., ListGroup)
-- Each component folder should have an `index.js` file with **named exports** of all the relevant components in the folder
-- Always prefer **default imports** between components in the same folder
-- Components that are not getting exported (to consumers) from
-  the `index.js` file should be in a subfolder named `InnerComponents`
+- Ensure the code is properly formatted and there are no linting errors. You can check by running project's linter with `yarn lint`.
+- Avoid using the bindMethods syntax for attaching methods to a class. Instead use [class properties.](https://babeljs.io/docs/en/babel-plugin-proposal-class-properties)
 - Storybooks for PatternFly 3
   - Provide a storybook with your component named on the parent component with a `.stories.js` suffix (e.g., `ListGroup.stories.js`)
   - When your stories contain multiple files, put them in a subfolder named `Stories`
   - `src/**/*.stories.js` and `src/**/Stories/` are excluded from the package build output
 - Documentation for PatternFly 4
-  - Provide documentation for your component with a single Markdown file. (e.g., `ListGroup.md`)
-    See how to write documentation in the [`react-docs` README](./packages/patternfly-4/react-docs/README.md)
-- When writing a component and you want to use the classnames package, be sure to import and name it `classNames`. For example - `import classNames from 'classnames'`
-- When destructuring or spreading expressions , use ...props as the variable name.
-- Exporting components from other libraries (without manipulating them)
-  to consumers is a common task, use:
-
-  ```js
-  # Badge/Badge.js
-  import { Badge } from 'react-bootstrap';
-  export default Badge;
-
-  # Badge/index.js
```

Comment:
Maybe update instead of removing?

Reply:
We link to the react-docs README now instead of duplicating the information twice.